dtfs-central-api/Dockerfile

```diff
@@ -9,9 +9,12 @@ ENV GITHUB_SHA=$GITHUB_SHA
 WORKDIR /app
 COPY package.json .
 COPY package-lock.json .
-RUN npm ci
+
+COPY libs libs
+COPY dtfs-central-api dtfs-central-api
+
+RUN npm run ci:all
 RUN npm cache clean --force
-COPY . .
 
 # Execute Script
-CMD ["npx", "ts-node", "src/index.ts"]
+CMD ["npx", "ts-node", "dtfs-central-api/src/index.ts"]
```

Comment:
Not entirely convinced with this path convention, when already inside the container Dockerfile.